pyproject.toml

```
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["poetry-core=1.1.15"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "rockstor"
+version = "4.5.1"
+description = "Btrfs Network Attached Storage (NAS) Appliance."
+homepage = "https://rockstor.com/"
+repository = "https://github.com/rockstor/rockstor-core"
+documentation = "https://rockstor.com/docs/"
+readme = "README.md"
+keywords = ["linux", "NAS", "appliance", "btrfs"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+]
+authors = [
+    "The Rockstor Project <support@linuxlines.com>",
+]
+maintainers = [
+    "Philip Paul Guyton <support@linuxlines.com>",
+]
+license = "GPL-2.0-or-later"
+exclude = [
+    {path = "test"},
+]
+
+# We use groups to clarify our dependencies (essentially labels):
+# https://python-poetry.org/docs/managing-dependencies/#dependency-groups
+[tools.poetry.dependencies]
```

I believe this needs to be without the "s":
```suggestion
[tool.poetry.dependencies]
```